runtime/development/src/lib.rs

```diff
@@ -1601,19 +1619,20 @@ construct_runtime!(
 		CrowdloanClaim: pallet_crowdloan_claim::{Pallet, Call, Storage, Event<T>} = 93,
 		CrowdloanReward: pallet_crowdloan_reward::{Pallet, Call, Storage, Event<T>} = 94,
 		PoolSystem: pallet_pool_system::{Pallet, Call, Storage, Event<T>} = 95,
-		Loans: pallet_loans::{Pallet, Call, Storage, Event<T>} = 96,
-		Permissions: pallet_permissions::{Pallet, Call, Storage, Event<T>} = 97,
-		CollatorAllowlist: pallet_collator_allowlist::{Pallet, Call, Storage, Config<T>, Event<T>} = 98,
-		Tokens: pallet_restricted_tokens::{Pallet, Call, Event<T>} = 99,
-		NftSales: pallet_nft_sales::{Pallet, Call, Storage, Event<T>} = 100,
-		Nfts: pallet_nft::{Pallet, Call, Event<T>} = 103,
-		Bridge: pallet_bridge::{Pallet, Call, Storage, Config<T>, Event<T>} = 101,
-		InterestAccrual: pallet_interest_accrual::{Pallet, Storage, Event<T>, Config<T>} = 102,
-		Keystore: pallet_keystore::{Pallet, Call, Storage, Event<T>} = 104,
-		Investments: pallet_investments::{Pallet, Call, Storage, Event<T>} = 105,
-		Rewards: pallet_rewards::{Pallet, Storage, Event<T>} = 106,
-		LiquidityRewards: pallet_liquidity_rewards::{Pallet, Call, Storage, Event<T>} = 107,
-		Connectors: pallet_connectors::{Pallet, Call, Storage, Event<T>} = 108,
+		PoolRegistry: pallet_pool_registry::{Pallet, Call, Storage, Event<T>} = 96,
```

Comment:
Renumbering is technically an API break (although the metadata update should handle it?)

Unless we are having issues with ordering of the pallets, I'd prefer to add the new one at the bottom

Reply:
It will also break decoding past events, errors and calls, as far as I can recall.